nearcore/src/lib.rs

```diff
@@ -511,9 +500,7 @@ pub fn start_with_config_and_synchronization(
     if let Some(db_metrics_arbiter) = db_metrics_arbiter {
         arbiters.push(db_metrics_arbiter);
     }
-    if let Some(partial_witness_arbiter) = partial_witness_arbiter {
-        arbiters.push(partial_witness_arbiter);
-    }
+    arbiters.push(partial_witness_arbiter);
```

Comment:
nit: you can add it directly where arbiters are created together with all the other non-optional arbiters